### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by llaville**
And also the README main page

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -58,6 +58,7 @@ Polyfills are provided for:
 - the `Stringable` interface introduced in PHP 8.0;
 - the `PhpToken` class introduced in PHP 8.0 when the tokenizer extension is enabled;
```
Comment (nicolas-grekas) [inline README.md:59]: can you also add this line to the readme of Php80 please?

Comment (llaville) [reply README.md:59]: `Stringable` interface is already  present in Php80 Readme ;-)  (https://github.com/symfony/polyfill/blame/main/src/Php80/README.md#L6) , but I'll add a link to PHP documentation (https://www.php.net/manual/en/class.stringable)

Comment (nicolas-grekas) [reply README.md:59]: I mean the line about PhpToken

Comment (llaville) [reply README.md:59]: FYI: I've added more links to PHP Documentation with  commit https://github.com/symfony/polyfill/pull/401/commits/58b6df77562437e4772ba8524db6819bb756df2a

**Comment by nicolas-grekas** [comment]
Thank you @llaville.

**Pull request merged**